gcp_variant_transforms/libs/vcf_header_definitions.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Google Inc.  All Rights Reserved.
```

Comment:
This class is a pure data object, and it is only used for `HeaderMerger`. I prefer to still include it in module `definitions_merger.py`. For one thing, the `libs` package usually defines functions of how to do something, it might not be expected to have an object definition here (and I can't find a proper package for it). Secondly, the way we resolve the conflicts is tied to how we define `Definition`, if it changes in the future, `definition_merger` is the only place we need to update. WDYT?